### PR TITLE
chore: Updated node v3 templates to node v18

### PR DESCRIPTION
**Pull request by Danwakeem**
## Description

- Updated all of the node templates users see when running the `sls` command to node 18
- Updated any aws sdk calls to use the v3 sdk
- Set up a consistent folder structure for the default templates
- Changed handler names to `handler` instead of something like `hello`
- Removed the comment about running lint before submitting a PR since it doesn't seem to be working or ran in a while 😎

### CODE REVIEW

File: aws-node-express-api/src/index.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ app.get("/", (req, res, next) => {
   });
```
Comment (medikoo) [inline aws-node-express-api/src/index.js:8]: Why is it moved to `src` ? There's no transpilation implied here, so I think there's no valid reason for that

**Review by medikoo — COMMENTED** [review]
Thanks @Danwakeem it looks great, I have just one concern

File: aws-node-express-api/src/index.js (hunk 1 above)
Comment (Danwakeem) [reply aws-node-express-api/src/index.js:8]: Yeah this is true 🤔 I can remove the src folder in the projects where I had created one.

Comment (medikoo) [reply aws-node-express-api/src/index.js:8]: Yes, I believe we should not add it

Comment (Danwakeem) [reply aws-node-express-api/src/index.js]: I just removed the `src` folder from all of the projects I added it to 👍

**Review by medikoo — APPROVED** [review]
Looks great 👍 

Technicaly in CJS `use strict` should be in every module, but I'll sort this out here with global setup of linter that's missing here.

**Comment by Danwakeem** [comment]
Ahh okay my bad I could have left that in there as well but I wasn't sure if it was necessary or not 🤦‍♂️

**Pull request merged**